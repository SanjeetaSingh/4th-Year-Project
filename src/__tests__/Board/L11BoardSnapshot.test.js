import renderer from 'react-test-renderer'

import Level11 from '../../Level11/L11-Board/L11-Board'

it('snapshot test level 11', () => {
    const tree =  renderer.create(<Level11 />).toJSON()
    expect(tree).toMatchSnapshot();
});