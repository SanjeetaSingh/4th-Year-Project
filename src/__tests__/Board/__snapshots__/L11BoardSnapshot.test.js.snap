// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test level 11 1`] = `
<div
  className="container"
>
  <div
    className="boardcontainer4"
  >
    <div
      className="board11"
    >
      <div
        className="box"
        data-col={0}
        data-row={0}
      >
        <img
          alt="d"
          class="image"
          id="cat"
          src="assets/cat.png"
        />
      </div>
      <div
        className="box"
        data-col={1}
        data-row={0}
      />
      <div
        className="box"
        data-col={2}
        data-row={0}
      >
        <img
          alt="f"
          class="bone"
          id="food"
          src="assets/food.png"
        />
      </div>
      <div
        className="box"
        data-col={0}
        data-row={1}
      >
        <img
          alt="f"
          class="bone"
          id="foodTwo"
          src="assets/food.png"
        />
      </div>
      <div
        className="box"
        data-col={1}
        data-row={1}
      />
      <div
        className="box"
        data-col={2}
        data-row={1}
      />
      <div
        className="box"
        data-col={0}
        data-row={2}
      >
        <img
          alt="d"
          class="image"
          id="dog"
          src="assets/dog.png"
        />
      </div>
      <div
        className="box"
        data-col={1}
        data-row={2}
      />
      <div
        className="box"
        data-col={2}
        data-row={2}
      />
    </div>
  </div>
  <div
    className="bottom-panel"
  >
    <div
      class="border"
    >
      <div
        class="level5Contain"
      >
        <h2
          class="h26"
        >
          Level 11:
        </h2>
        <div
          class="speech6"
        >
          Your aim for this level is to help the dog get to the both of the food bowls, you have to use the while command to limit the amount of times you repeat the commands. You have use the if statement to avoid the moving cat and reach the food. You have to use the same number or less number of commands mentioned in the top right corner.
          <br />
           Good luck! 
          <br />
           (To learn more about while and to understand the while loop condition and if statement click the i below)
        </div>
        <div
          class="borderPanel"
        >
          <div
            class="toppart6"
          >
            <div
              class="containing"
            >
              <p
                class="number"
                data-testid="counter"
                id="count"
              >
                0/5
              </p>
            </div>
            <p
              class="titles"
            >
              Enter Sequence:
            </p>
            <p
              class="lists"
              id="action"
            />
          </div>
        </div>
        <div
          class="outside6"
        >
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Left
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Up
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Down
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Right
          </button>
          <div
            class="buttons-wrapper5"
          >
            <button
              class="seemingly-inner-button"
              disabled={false}
              onClick={[Function]}
            >
              <i
                class="fa fa-info"
              />
            </button>
            <button
              class="button"
              disabled={false}
              onClick={[Function]}
              type="submit"
            >
              While
            </button>
          </div>
          <div
            class="buttons-wrapper5"
          >
            <button
              class="seemingly-inner-button"
              disabled={false}
              onClick={[Function]}
            >
              <i
                class="fa fa-info"
              />
            </button>
            <button
              class="button"
              disabled={false}
              onClick={[Function]}
            >
              If Statement
            </button>
          </div>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Bark
          </button>
          <button
            class="buttonClear"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Clear
          </button>
          <button
            class="buttonSub"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
             Submit
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
